packages/repack/src/modules/ScriptManager/__tests__/ScriptManager.test.ts

```diff
@@ -832,6 +839,43 @@ describe('ScriptManagerAPI', () => {
     spy.mockReset();
     spy.mockRestore();
   });
+
+  it('should refetch failed script', async () => {
+    jest.useFakeTimers({ advanceTimers: true });
+    const spy = jest.spyOn(NativeScriptManager, 'loadScript');
+
+    spy.mockImplementation(
+      (_scriptId: string, _scriptConfig: NormalizedScriptLocator) =>
+        Promise.reject({ code: 'NetworkFailed' })
+    );
```

Comment:
maybe we can get away with `.mockRejectedValueOnce` ?

Reply:
i will try